tabpy-server/server_tests/test_pwd_file.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+import unittest
+from argparse import Namespace
+from tempfile import NamedTemporaryFile
+
+from tabpy_server.app.app import TabPyApp
+from tabpy_server.app.ConfigParameters import ConfigParameters
+
+try:
+    from unittest.mock import patch, call
+except ImportError:
+    from mock import patch, call
+
+
+class TestPasswordFile(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(TestPasswordFile, self).__init__(*args, **kwargs)
+        self.config_file = None
+        self.pwd_file = None
+        self.cwd = os.getcwd()

```

Comment:
I would advise against using the current directory as it limits where the test suite can be executed from. Not a huge deal, but really only downsides to doing this

Reply:
agreed